src/java/picard/analysis/CollectGcBiasMetrics.java

```diff
@@ -59,15 +59,54 @@
  */
 @CommandLineProgramProperties(
-        usage = "Tool to collect information about GC bias in the reads in a given BAM file. Computes" +
-                " the number of windows (of size specified by WINDOW_SIZE) in the genome at each GC%" +
-                " and counts the number of read starts in each GC bin.  What is output and plotted is" +
-                " the \"normalized coverage\" in each bin - i.e. the number of reads per window normalized" +
-                " to the average number of reads per window across the whole genome..\n",
-        usageShort = "Collects information about GC bias in the reads in the provided SAM or BAM",
+        usage = CollectGcBiasMetrics.USAGE_SUMMARY + CollectGcBiasMetrics.USAGE_DETAILS,
+        usageShort = CollectGcBiasMetrics.USAGE_SUMMARY,
         programGroup = Metrics.class
 )
 public class CollectGcBiasMetrics extends SinglePassSamProgram {
+    static final String USAGE_SUMMARY = "Collects information regarding GC bias, from a SAM/BAM input file.  ";
+    static final String USAGE_DETAILS = "Tool that collects information about the proportions of guanine (G) and cytosine (C)" +
+            " nucleotides in a sample.  Regions of high and low G + C content have been shown to interfere with mapping/aligning," +
+            " ultimately leading to low read depth and fragmented genome assemblies, a phenomenon known as \"GC bias\".  " +
+            "Detailed information on the effects GC-bias on NGS data can be found at DOI: 10.1371/journal.pone.0062856/.<br /><br />." +
+            "" +
+            "For each run, the corresponding reference sequence is divided into bins or windows based on the percentage of G + C" +
+            " content ranging from 0 - 100%.  The percentages of G + C are determined from a defined length of sequence, the default " +
+            "value is set at 100 bases.   " +
+            "Although the mean of the distribution will vary among organisms, human DNA has a mean GC-content of 40%, suggesting " +
+            "preponderance of AT-rich regions.  <br /><br />" +
+
+            "GC bias is calculated and output as both summary (optional) and detailed metrics (required).  " +
+            "The GcBiasSummaryMetrics provides high-level metrics that capture run-specific bias information including" +
+            " WINDOW_SIZE, ALIGNED_READS, TOTAL_CLUSTERS, AT_DROPOUT, and GC_DROPOUT.  While WINDOW_SIZE refers to the" +
+            "numbers of bases used for the distribution (see above), the ALIGNED_READS and" +
+            " TOTAL_CLUSTERS are the total number of aligned reads and the total number of reads (after filtering) " +
+            "produced in a run.   In addition, the tool produces both AT_DROPOUT and GC_DROPOUT metrics, which indicate the percentage of" +
+            " reads dropped from an analysis due to the inability to map to the reference as result of excessively" +
+            " GC-poor or GC-rich regions respectfully. <br /><br />" +
```

Comment:
The AT_DROPOUT and GC_DROPOUT isn't only looking at GC-poor or GC-rich regions, it also looks at the 40-60% gc content range, which is balanced. If the GC content is 50% or less the dropout gets added as AT dropout, and if it's over 50% it is added to GC dropout.